nginx-redirects.conf

```
@@ -325,6 +325,7 @@ server {
 
 server {
     include /etc/nginx/ssl.default.conf;
+    include /etc/nginx/api-proxy.conf;
     server_name floatingforests.org www.floatingforests.org;
     return 301 https://www.zooniverse.org/projects/zooniverse/floating-forests;
```

You'll need to wrap this in a `location /` block or it will take precedence over the API locations:

```suggestion
    location / {
        return 301 https://www.zooniverse.org/projects/zooniverse/floating-forests;
    }
```